packages/ui/src/domainModules/collectionMammals/components/MammalManager/MainColumn/RecordForm/formParts/PhysicalObjectsAccordion/PhysicalObjectTitle.js

```diff
@@ -80,14 +70,14 @@ function PhysicalObjectTitle({
     curatorialAssessments.length &&
     curatorialAssessments[curatorialAssessments.length - 1]
 
-  const { group, name, parent } = storageLocation || {}
+  const storageLocationText =
+    storageLocation && extractNameWithFirstLevelParent(storageLocation)
+
   return (
     <ThreeColumnGrid
       center={
         storageLocation && (
-          <span style={{ fontWeight: 'normal' }}>
-            {`${name} [${group} in ${getFirstLevelParent(parent)}]`}
-          </span>
+          <span style={{ fontWeight: 'normal' }}>{storageLocationText}</span>
```

Comment:
since you have the check for `storageLocation &&` in line 79, I think it's fine to just do `{extractNameWithFirstLevelParent(storageLocation)}` here, and remove line 73-74